src/models/product.py

```diff
@@ -42,6 +42,74 @@ def _missing_(cls, value):
         return None
 
 
+@unique
+class Category(StrEnum):
+    """
+    The normalised category of the product.
+    """
+
+    VIANDE = "Viandes"
+    POISSON = "Poissons et fruits de mer"
+    LAITIER = "Œufs et produits laitiers"
+    LEGUMINEUSE = "Légumineuses"
+    SOJA = "Produits à base de soja"
+    ASSIMILE = "Légumes et assimilés"
+    CEREALE = "Céréales et pseudo-céréales"
+    NOIX = "Noix et graines"
+    POUDRE = "Poudres protéinées"
+    ALTERNATIVE = "Alternatives végétales"
+
+    @classmethod
+    def _missing_(cls, value):
+        """
+        Invoked when the value is not found in the enum. It is used here to
+        accept values in a case-insensitive way.
+
+        See https://docs.python.org/3/library/enum.html#enum.Enum._missing_.
+        """
+
+        value = value.upper()
+
+        for member in cls:
+            if member.value.upper() == value:
+                return member
+
+        return None
+
+
+@unique
+class Department(StrEnum):
+    """
+    The store main department of the product.
+    """
+
+    LAITIER = "Produits laitiers, oeufs, fromages"  # cn01
+    VIANDE = "Boucherie, volaille, poissonnerie"  # cn02
+    CHARCUTERIE = "Charcuterie, traiteur"  # cn12
+    FRAIS = "Marché frais"  # cb19
+    EPICERIE = "Epicerie salée"  # cn06
+    SUCRE = "Epicerie sucrée"  # cn05
+    FRUIT_LEGUME = "Fruits, légumes"  # cn03
+    SURGELE = "Surgelés"  # cn04
```

Comment:
These are the store departments of Auchan, not our "Aliment" sub-category.

We need to implement our categories and sub-categories as database columns, and then extract and cross match in each scraper the distributor's own categories and ours.

I recommend moving this enumeration in `src/scraper_auchan`.